### PR TITLE
doc: update kcli information

### PR DESCRIPTION
**Pull request by karmab**
Updates the doc information related to kcli

Signed-off-by: karmab karimboumedhel@gmail.com

### CODE REVIEW

File: doc/dev/cephadm/developing-cephadm.rst [hunk 1]
```diff
@@ -145,69 +145,65 @@ virtualization providers (libvirt, KubeVirt, oVirt, OpenStack, VMware vSphere,
...
-install Ceph clusters using Ceph ansible or cephadm, lets create our first Ceph
+install Ceph clusters using Ceph ansible or cephadm, so let's create our first Ceph
 cluster using cephadm::
 
-# kcli2 create plan -u https://github.com/karmab/kcli-plans/blob/master/ceph/ceph_cluster.yml
+# kcli create plan -u https://github.com/karmab/kcli-plans/blob/master/ceph/ceph_cluster.yml
 
 This will create a set of three vms using the plan file pointed by the url.
-After a few minutes (depend of your laptop power), lets examine the cluster:
+After a few minutes (dependinf on your laptop power), let's check the cluster:
```
Comment (mgfritch) [inline doc/dev/cephadm/developing-cephadm.rst:206]:
```suggestion
After a few minutes, let's check the cluster:
```

File: doc/dev/cephadm/developing-cephadm.rst [hunk 2]
```diff
@@ -145,69 +145,65 @@ virtualization providers (libvirt, KubeVirt, oVirt, OpenStack, VMware vSphere,
...
-   suggest you to "save" and use your own kcli image.
-   Why?: kcli is alive and it changes (and for the moment only exists one tag ...
-   latest). Because we have more than enough with the current functionality, and
-   what we want is overall stability,
-   we suggest to store the kcli image you are using in a safe place and update
-   your kcli alias to use your own image.
+   suggest you stick to a given container tag and update your kcli alias.
+   Why? kcli is an alive progress and works in a rolling release model. 
+   Because we have more than enough with the current functionality, and
+   what we want is overall stability.
```
Comment (mgfritch) [inline doc/dev/cephadm/developing-cephadm.rst:183]:
```suggestion
   suggest you stick to a given container tag and update your kcli alias.
   Why? kcli uses a rolling release model and sticking to a specific 
   container tag will improve overall stability.
```

this section is somewhat unclear

**Comment by mgfritch** [comment]
ping @jmolmo

**Review by jmolmo — APPROVED** [review]
Thanks!

**Comment by sebastian-philipp** [comment]
ping  @karmab want to change your PR following the feedback?

**Comment by karmab** [comment]
> ping @karmab want to change your PR following the feedback?

i think i did?

**Comment by sebastian-philipp** [comment]
jenkins test signed

**Pull request merged**